handbook/product/index.md

```diff
@@ -6,28 +6,24 @@ The product team is newly growing, and consists of all product managers and prod
 
 ### Members
 
-- [Product managers](roles.md#product-manager)
+- [Product managers](roles/product_manager.md)
   - [Christina Forney](../../../company/team/index.md#christina-forney-she-her)
   - [Pooja Jain](../../../company/team/index.md#pooja-jain-she-her)
   - [M.A.C](../../../company/team/index.md#todo) starting 2020-08-24
-- [Product designers](roles.md#product-designer)
+- [Product designers](roles/product_designer.md)
   - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
   - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
   - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
-- [Developer communications](roles.md#developer-communications)
-  - [Kimberly Hopson](../../company/team/index.md#kimberly-hopson-she-her)
+- [Technical Writers](roles/technical_writer.md)
+  - [Kimberly Hopson](../../../company/team/index.md#kimberly-hopson-she-her)
 
-### [Roles and responsibilites](roles.md)
+### [Roles](roles/index.md)
```

Comment:
The roles and hiring sections felt redundant, so I combined them. They still feel a bit redundant to the Members section above, and in order to not have broken links I needed to create the redundant roles/index.md that simply links to the roles pages. If you have ideas how we can clean this up while keeping it top level, I'd love to hear suggestions!

Reply:
I made this a little better by linking to the docs areas in the Members section, and am now only linking to the roles pages here. Feedback still appreciated!